src/main.tsx

```diff
@@ -1,6 +1,4 @@
 import './installSesLockdown';
-import 'ses';
```

Comment:
why is this necessary? did something change?

Reply:
`installSesLockdown` already imports `ses` so it was redundant

Reply:
Very well. I wonder if we should even import `ses` there.  This looks like Endo's general lockdown handler: https://github.com/endojs/endo/blob/master/packages/lockdown/pre.js

Then there'd be less magic, because `lockdown` would come from an import instead of as a side effect.
```js
import { lockdown } from '@endo/lockdown';
```

@kriskowal is that the current best practice?

Reply:
Oh yea that works